packages/react-router/lib/hooks.tsx

```diff
@@ -19,6 +19,7 @@ import {
   parsePath,
   resolveTo,
   warning,
+  UNSAFE_getPathContributingMatches as getPathContributingMatches,
```

Comment:
Removed the duped function and exported it as a private function from `@remix-run/router`